src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -137,10 +194,36 @@ protected override void BeginProcessing()
         protected override void ProcessRecord()
         {
             bool result = true;
+            string resolvedPath = string.Empty;
+            string jsonToParse = string.Empty;
+
+            if (Json != null)
+            {
+                jsonToParse = Json;
+            }
+
+            if (Path != null)
+            {
+                resolvedPath = PathUtils.ResolveFilePath(Path, this, _isLiteralPath);
+
+                if (!System.IO.File.Exists(resolvedPath))
+                {
+                    ItemNotFoundException exception = new(
+                        resolvedPath,
+                        "PathNotFound",
+                        SessionStateStrings.PathNotFound);
+
+                    ThrowTerminatingError(new ErrorRecord(
+                        exception.ErrorRecord,
+                        exception));
+                }
+
+                jsonToParse = File.ReadAllText(resolvedPath);
+            }
 
             try
             {
-                var parsedJson = JToken.Parse(Json);
+                var parsedJson = JToken.Parse(jsonToParse);
```

Comment:
Maybe return early in case `jsonToParse` is empty or contains white spaces only?

Reply:
Added condition to check for null or whitespace